src/Bridge/Symfony/Bundle/Command/SwaggerCommand.php

```diff
@@ -54,9 +57,11 @@ public function __construct(NormalizerInterface $documentationNormalizer, Resour
     protected function configure()
     {
         $this
-            ->setName('api:swagger:export')
-            ->setDescription('Dump the Swagger 2.0 (OpenAPI) documentation')
+            ->setName('api:openapi:export')
```

Comment:
I did this change as I remembered I saw it in a ticket somewhere quite a long time ago.